data/gowthae.json

```diff
@@ -0,0 +1,36 @@
+
+{
+  "name": "Gowtham E",
+  "displayStatsPublic":true,
+  "type": "personal",
+  "bio": "I'm a Software Developer",
+  "avatar": "https://github.com/gowthae.png",
```

Comment:
I believe this field is no longer required. The avatar is automatically fetched from your profile.
```suggestion
```